src/riskmatrix/views/risk_assessment.py

```diff
@@ -18,7 +20,7 @@
 from riskmatrix.wtform.validators import Disabled
 
 
-from typing import Any, TYPE_CHECKING
+from typing import Any, Generator, TYPE_CHECKING
```

Comment:
```suggestion
from typing import Any, TYPE_CHECKING
```
Don import ABCs from the typing module, import them inside the `if TYPE_CHECKING` block from `collections.abc`.

In this case `Iterator` is also the more appropriate protocol than `Generator`. `Generator` only makes sense if you use at least one of the two other type params.